components/layout/inline.rs

```diff
@@ -1798,6 +1823,16 @@ fn inline_contexts_are_equal(inline_context_a: &Option<InlineFragmentContext>,
     }
 }
 
+fn side_borders_and_paddings_are_definitely_zero(style: &ServoComputedValues) -> bool {
+    let border = style.get_border();
+    let padding = style.get_padding();
+
+    border.border_left_width == Au(0) &&
+    border.border_right_width == Au(0) &&
+    padding.padding_left.is_definitely_zero() &&
+    padding.padding_right.is_definitely_zero()
```

Comment:
`is_definitely_zero` is, as the name implies, not an exhaustive test for whether the "used property" of a value is zero. The spec asks for the used value.

How hard would it be to get the used value here?